vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -110,9 +112,19 @@ class HomeActivityViewModel @AssistedInject constructor(
         checkSessionPushIsOn()
         observeCrossSigningReset()
         observeAnalytics()
+        observeReleaseNotes()
         initThreadsMigration()
     }
 
+    private fun observeReleaseNotes() {
+        releaseNotesPreferencesStore.appLayoutOnboardingShown.onEach { isAppLayoutOnboardingShown ->
+            if (!isAppLayoutOnboardingShown) {
+                _viewEvents.post(HomeActivityViewEvents.ShowReleaseNotes)
+                releaseNotesPreferencesStore.setAppLayoutOnboardingShown(true)
```

Comment:
I would move this line at above the previous one, just to be 100% sure that the viewModel scope is not cancelled before we set this boolean. Action `HomeActivityViewEvents.ShowReleaseNotes` will start another Activity.